lib/network_manager.go

```diff
@@ -404,6 +410,11 @@ func (nm *NetworkManager) processOutboundConnection(conn Connection) (*RemoteNod
 		return nil, fmt.Errorf("NetworkManager.handleOutboundConnection: Connection is not an outboundConnection")
 	}
 
+	if !addrmgr.IsRoutable(oc.address) {
```

Comment:
I thought we were going to delete this. Wasn't it commented out in the last revision? If we have it it could mess up nodes running within datacenters.